lib/screens/stake_screen/stake_screen.dart

```diff
@@ -0,0 +1,340 @@
+import 'dart:ui';
+
+import 'package:deus/core/widgets/dark_button.dart';
+import 'package:deus/core/widgets/filled_gradient_selection_button.dart';
+import 'package:deus/core/widgets/selection_button.dart';
+import 'package:deus/core/widgets/svg.dart';
+import 'package:deus/statics/my_colors.dart';
+import 'package:deus/statics/styles.dart';
+import 'package:flutter/material.dart';
+
+enum StakesStates {
+  hasToApprove,
+  pendingApprove,
+  canStake,
+  stakePendingApprove,
+  stakeApproved
+}
+
+class StakeScreen extends StatefulWidget {
+  static const url = '/stake';
+
+  @override
+  _StakeScreenState createState() => _StakeScreenState();
+}
+
+class _StakeScreenState extends State<StakeScreen> {
+  final String address = '0x3a6dabD6B5C75291a3258C29B418f5805792a87e';
+
+  final double apy = 99.93;
+
+  final balance = 13.234;
+
+  final gradient = MyColors.blueToGreenGradient;
+
+  StakesStates _stakeState = StakesStates.hasToApprove;
+  bool _showToast = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color(MyColors.Background),
+      body: SafeArea(
+        child: SingleChildScrollView(
+          padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 15),
+          child: SizedBox(
+            height: MediaQuery.of(context).size.height-80,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                _buildHeaderAddress(),
+                SizedBox(
+                  height: 23,
+                ),
+                _buildBackButton(),
+                SizedBox(
+                  height: 20,
+                ),
+                Text(
+                  'Stake your sDEA',
+                  style: TextStyle(fontSize: 25),
+                ),
+                SizedBox(
+                  height: 8,
+                ),
+                Text(
+                  '$apy% APY',
+                  style: TextStyle(fontSize: 20),
+                ),
+                SizedBox(
+                  height: 20,
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(left: 15),
+                  child: Text(
+                    'Balance: $balance',
+                    style: MyStyles.lightWhiteSmallTextStyle,
+                  ),
+                ),
+                SizedBox(
+                  height: 5,
```

Comment:
So SizedBoxes würde ich tendenziell immer nur als eine Zeile lassen, damit es weniger Platzeinnimmt.
- [ ] Benutz Alt-Taste und klick hinter alle Kommas bei den heights, dann Backspace => alle Kommas sind gleichzeitig weg
- [ ] danach Strg + Alt + L zum Reformattieren und schon ist es nur noch in einer Zeile.

Reply:
nur durch Strg + Alt + L teilt es sich erst so auf ^^

Reply:
Oh, das ist ein wichtiger Hinweis! Also wenn du das Komma gelöscht hast.

Weil dass das dreizeilig wird, wenn das Komma da ist, ist mir schon klar. Deswegen ja der erste Schritt. Aber bei dir klingt das so, als wird es automatisch hinzugefügt?! Ist das richtig? Wenn ja, schreib mir auf Slack und schreib es in Notion in unserer [Feedbackseite](https://www.notion.so/davesspace/Ideen-Bookmarks-Feedback-74675443f1164c4daf98666c44d64579) in die Herausforderungen.

Reply:
- [ ] Lösung für Formatierung finden